content/swagger/resources/measurement_families/routes/measurement_families.yaml

```diff
@@ -163,3 +163,46 @@ get:
       $ref: "#/responses/403Error"
     406:
       $ref: "#/responses/406Error"
+patch:
+  summary: Update/create several measurement families
+  operationId: "patch_measurement_families"
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  description: This endpoint allows you to update and/or create several measurement families at once.
+  x-body-by-line: Contains several lines, each line is a measurement family in JSON standard format
+  parameters:
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/MeasurementFamily"
+        x-examples:
+          x-example-1:
+            '{"code":"Illuminance","standard_unit_code":"LUX","units":[{"code": "LUX","convert_from_standard":[{"operator": "mul", "value": "1"}], "symbol": "lx"}]}'
+          x-example-2:
+            '{"code":"Binary","standard_unit_code":"BIT","units":[{"code": "BIT","convert_from_standard":[{"operator": "mul", "value": "1"}], "symbol": "b"}]}'
+          x-example-3:
+            '{"code":"Area","units":[{"code": "ARPENT","convert_from_standard":[{"operator": "mul", "value": "3418.89"}], "symbol": "arpent"}]}'
```

Comment:
there should be all the measurement family information here :/

Reply:
c'est ce qu'on avait testé tout à l'heure, la famille existe et je renvois juste une unité pour changer la standard

Reply:
seen IRL